### DISCUSSION TITLE
An introduction to Plenti

### DISCUSSION THREAD

**Discussion by jamestagal** [plentico/plenti#311] [Show and tell]
Hi all,
I recorded an unscripted video introduction to Plenti, see linked below, for a couple of fellow Australians in a Discord community I am involved in. So it was meant to be fairly informal with the purpose to give them a quick overview.  I thought I would post it as well even though it is pretty rough and ready! I tried my best to explain the basics, the overall architecture, file structure and CMS and how components work with the content source. Also showed creating a new Plenti site. I only scratched the surface with other concepts and functionality within Plenti but will work towards making a concise video of Plenti and include all its features. 

https://youtu.be/XE51M18W2qM

**Comment by jimafisk** [comment]
This video is great @jamestagal, thanks for sharing it here!
